regression/goto-analyzer/dependence-graph10/test.desc

```diff
@@ -0,0 +1,25 @@
+CORE
+main.c
+--dependence-graph --show
+activate-multi-line-match
+EXIT=0
+SIGNAL=0
+// First assignment has a control dependency on the if
+\/\/ ([0-9]+).*\n.*IF.*i < 10.*THEN(.*\n)*Control dependencies: \1\n(.*\n){2,3}.*a = 1
+// Second assignment has a control dependency on the if
+\/\/ ([0-9]+).*\n.*IF.*i < 10.*THEN(.*\n)*Control dependencies: \1\n(.*\n){2,3}.*a = 2
+--
+^warning: ignoring
+--
+The two regexes above match output portions like shown below (with <N> being a
```

Comment:
This explanation is great, though I was hoping to have both lines laid about explicitly